libindy/tests/anoncreds.rs

```diff
@@ -384,31 +370,44 @@ mod high_cases {
         fn prover_get_claims_works_for_filter_by_schema() {
             let (wallet_handle, _) = AnoncredsUtils::init_common_wallet();
 
-            let claims = AnoncredsUtils::prover_get_claims(wallet_handle, &format!(r#"{{"schema_seq_no":{}}}"#, GVT_SEQ_NO)).unwrap();
+            let claims = AnoncredsUtils::prover_get_claims(wallet_handle, &format!(r#"{{"schema_key":{}}}"#, &AnoncredsUtils::gvt_schema_key_json())).unwrap();
             let claims: Vec<ClaimInfo> = serde_json::from_str(&claims).unwrap();
 
             assert_eq!(claims.len(), 2);
-            assert_eq!(claims[0].schema_seq_no, GVT_SEQ_NO);
-            assert_eq!(claims[1].schema_seq_no, GVT_SEQ_NO);
+            assert_eq!(claims[0].schema_key, AnoncredsUtils::gvt_schema_key());
+            assert_eq!(claims[1].schema_key, AnoncredsUtils::gvt_schema_key());
+        }
+
+
+        #[test]
+        fn prover_get_claims_works_for_filter_by_part_of_schema() {
+            let (wallet_handle, _) = AnoncredsUtils::init_common_wallet();
+
+            let claims = AnoncredsUtils::prover_get_claims(wallet_handle, r#"{"schema_key":{"name":"gvt"}}"#).unwrap();
```

Comment:
Please add tests to make sure that we can use any part of the Schema (name or verion or did)